src/x11/x.c

```diff
@@ -485,7 +490,7 @@ static void x_win_setup(void)
                                    (0xffffffff / 100)));
 
         if (settings.f_mode != FOLLOW_NONE) {
-                long root_event_mask = FocusChangeMask | PropertyChangeMask;
+                long root_event_mask = FocusChangeMask | PropertyChangeMask | SubstructureNotifyMask;
```

Comment:
This will introduce a stupid bug. We have to listen to the `SubstructureNotifyMask` anyways, not conditioned to the follow mode.

Reply:
Good catch, fixed!